chore: improve docker configuration (#464)
leav-solutions/leav-engine: README.md

Then, all you have to do is:
```
cd docker
docker compose up -d
```
This will start LEAV-Engine in development mode.

⚠️ **Docker v.20.10** or higher is required to run the core.

### Profiles

In development mode LEAV-Engine is started without certain services. To do so we use Docker profiles that allow us to define different configurations for your containers based on specific use cases or environments.

Here are some profiles that you can use:

#### Logs

> This profile can be used to retrieve application logs and starting `elasticsearch`.

Reviewer: **suggestion:** 
```suggestion
This profile can be used to retrieve application logs.
Services concerned:  `elasticsearch`, `logs-collector`
```

```bash
docker compose --profile logs up
```

#### Debug

> This profile can be used if you need `redis-commander` or `kibana`.

Reviewer: **suggestion:** 
```suggestion
This profile can be used if you need to get more infos about redis or elasticsearch
Services concerned: `redis-commander`, `kibana`.
```

```bash
docker compose --profile debug up
```

#### Automate

> This profile can be used if you need all services related to the automate.

Reviewer: **suggestion:** 
```suggestion
This profile can be used if you need all services related to the automate (events scan, previews generation).
Services concerned: `files-manager`, `preview-generator`, `automate-scan`, `sync-scan`
```

```bash
docker compose --profile automate up
```

> 

Reviewer: **suggestion:** 
```suggestion

More infos about profiles can be found [here](https://docs.docker.com/compose/profiles/)
```
